huaweicloud/services/cts/resource_huaweicloud_cts_data_tracker.go

```diff
@@ -379,3 +382,37 @@ func updateDataTrackerStatus(c *client.CtsClient, name, status string) error {
 	_, err := c.UpdateTracker(&statusReq)
 	return err
 }
+
+func resourceCTSDataTrackerImportState(_ context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	trackerName := d.Id()
+
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+	ctsClient, err := cfg.HcCtsV3Client(region)
+	if err != nil {
+		return []*schema.ResourceData{d}, fmt.Errorf("error creating CTS client: %s", err)
+	}
+
+	trackerType := cts.GetListTrackersRequestTrackerTypeEnum().DATA
+	listOpts := &cts.ListTrackersRequest{
+		TrackerName: &trackerName,
+		TrackerType: &trackerType,
+	}
+
+	response, err := ctsClient.ListTrackers(listOpts)
+	if err != nil {
+		return []*schema.ResourceData{d}, fmt.Errorf("error retrieving CTS data tracker")
+	}
+
+	if response.Trackers == nil || len(*response.Trackers) == 0 {
+		d.SetId("")
+		return []*schema.ResourceData{d}, fmt.Errorf("cannot retrieve CTS data tracker %s", trackerName)
```

Comment:
return []*schema.ResourceData{d}, fmt.Errorf("the CTS data tracker (%s) dose not exist", trackerName)

Reply:
fixed.